pkg/controller/jobs/raycluster/raycluster_controller_test.go

```diff
@@ -145,25 +145,26 @@ func TestReconciler(t *testing.T) {
 					Obj(),
 			},
 			wantWorkloads: []kueue.Workload{
-				*utiltesting.MakeWorkload("a", "ns").Finalizers(kueue.ResourceInUseFinalizerName).
+				*utiltesting.MakeWorkload("a", "ns").
 					Finalizers(kueue.ResourceInUseFinalizerName).
-					PodSets(kueue.PodSet{
-						Name:  "head",
-						Count: int32(1),
-						Template: corev1.PodTemplateSpec{
-							Spec: corev1.PodSpec{
-								RestartPolicy: corev1.RestartPolicyNever,
-								Containers: []corev1.Container{
-									{
-										Name: "head-container",
-										Resources: corev1.ResourceRequirements{
-											Requests: make(corev1.ResourceList),
+					PodSets(
```

Comment:
Ah, ok, there are not too many formatting changes here, so it may stay, but in general we try to do formatting in a dedicated PR, to minimize distractions.